htlcswitch/mailbox_test.go

```diff
@@ -548,8 +548,20 @@ func TestMailOrchestrator(t *testing.T) {
 
 	// First, we'll create a new instance of our orchestrator.
 	mo := newMailOrchestrator(&mailOrchConfig{
+		fetchUpdate: func(sid lnwire.ShortChannelID) (
+			*lnwire.ChannelUpdate, error) {
+			return &lnwire.ChannelUpdate{
+				ShortChannelID: sid,
+			}, nil
+		},
+		forwardPackets: func(_ chan struct{},
+			pkts ...*htlcPacket) chan error {
+			errChan := make(chan error)
+			close(errChan)
```

Comment:
nit: Maybe add a comment explaining why the `errchan` is closed immediately.

Reply:
done